isign/bundle.py

```diff
@@ -152,6 +174,13 @@ def sign(self, signer):
         executable = self.signable_class(self, self.get_executable_path())
         executable.sign(self, signer)
 
+        """ Making the main executable file (executable) """
+        if not is_app_Executable(self.get_executable_path()):
```

Comment:
I don't understand why the binary wouldn't have been executable. Are you saying that in your original app, the binary was executable, but in the resigned app, it was not executable?